notes/meeting13.md

```diff
@@ -0,0 +1,145 @@
+
+### 2. Tribute to talk
+
+(Eric) There isn't much, but the non-native touchpad for Keycard is also implemented, so now we have two implementations of it. Let's rather merge the two for Keycard and TtT instead of building others.
+
+(Jarrad) Barry has been doing economic modelling, and as TtT is currently implemented, it's very transactional. There might be a better way to adjust it: instead of putting up X tokens and having them relieved on intitial contact, it might be better to have both parties stake X SNT to each other until their relationship breaks down, i.e. only withdraw SNT at the time of "blocking" someone. More beneficial i.t.o. of locking up tokens in the network (relationships last longer than transactions), and i.t.o of spam mechanisms.
+
+(Ricardo) looks like a visibility staking mechanism. Was thinking of something similar for the public chats and governance of moderators. There are private, group and public chats: so is Tribute to Talk the correct name? TtT should replace Patreon - content creators are mad about patreon burning people due to political bias and this is an opportunity for Status to build uncensorable funding mechanisms. Problem on content-creator approach is in using SNT, which is prone to volatility. Perhaps we can do immediate conversions to stable coins for this kind of problem more generally in the future.
+
+### 3. DApp Store
+
+(Ricardo) Developed ranking systems with a contract that feeds the system with ranking of DApps, which we can use in tandem with the sticker market, potentially. Not sure how to work with the bonding curves and whether that's worthwhile, so haven't done anything there.
+
+(Andy) 1st Design Review delayed by changes to navigation tab in app. Blocked until that is resolved.
+
+### 4. Teller Network
+
+(Iuri) First PoC done with a whole lot of UI changes to make it more usable. We are redoing a few more UI pieces so that we can get to a fully-working DApp which can support multiple users.
+
+### 5. Network Incentivization
+
+(Ricardo) This should be multiple things, not one.
+
+(Oskar) It would be best to discuss further in the swarm calls the exact details of that if Andrea is not around. [link to GH or calendar invite?]
+
+### 6. Protocol
+
+(Oskar) Will meet at FCC and please ping me if you want to talk with specific people there.
+
+### 7. DAO
+
+(Ricardo) Research on topic democracy: changing from sub-topics to have sub-democracies, which are more secure. The problem is: to include new actions/topics I would have to pass a quorum. So now making it such that we can create a new democracy each time, with parent and child democracies that allow you to change ome properties without having to reach a supermajority each time.
+
+We can create propoals without democracies right now for testing, and that's where we are.
+
+## Better PR Process
+
+Document [here]() and in Discuss.
```

Comment:
https://notes.status.im/C5pj8g7gQOu9Wo8PtDZsMw# and https://discuss.status.im/t/better-pull-requests-process/1044

Reply:
fixed